src/src/tx_main.cpp

```diff
@@ -357,7 +357,7 @@ void ICACHE_RAM_ATTR SetRFLinkRate(uint8_t index) // Set speed of RF link (hz)
 
   DBGLN("set rate %u", index);
   hwTimer.updateInterval(ModParams->interval);
-  Radio.Config(ModParams->bw, ModParams->sf, ModParams->cr, GetInitialFreq(), ModParams->PreambleLen, invertIQ, ModParams->PayloadLength);
+  Radio.Config(ModParams->bw, ModParams->sf, ModParams->cr, GetInitialFreq(), ModParams->PreambleLen, invertIQ, ModParams->PayloadLength, ModParams->interval * 95 / 100);
```

Comment:
I don't think 95% of the interval should be used since that can still receive packets that aren't lined up in our slot. It isn't the case currently, but the interval can be much longer than the TOA (over 2x longer with 500Hz FLRC) which can lead to more spurious receptions.

If a packet is being received, the timeout is canceled (at least on SX1280) so this really just needs to be our dead time + a little bit for packet reception to start (somewhere in the `interval - TOA + 0.25 * TOA` ballpark). Without getting too mathy about it, I think just using the TOA is a good substitute? Ah crap that doesn't work with FLRC either. `interval - TOA + TOA / 2` aka `interval - (TOA / 2)`?

Reply:
What about when the loop isn't locked, it would be beneficial to have it as wide as possible because there's no certainty it'll line up. Maybe we need it to switch over after the PPL locks? Ah but then what about when telemetry if off. I saw as long as the RX doesn't overlap an interval it doesn't matter if get a spurious packet which the CRC/matching will reject. 

I know that the datasheet says that timeout is cancelled after having detected a preamble, but I found this NOT to be the case in my testing. If I set the timeout to 1/2 the packet interval I wasn't getting anything through. I don't know why, maybe because we're using implicit head mode?

Reply:
The RX operates in continuous mode so the timeout is not an issue for it syncing. I mentioned it in another comment but it might be better to switch over to a timeout once locked, but then we're just going to catch the TIMEOUT ISR and put it back into receive mode (since we have to be in RX mode before the TOCK), correct? I don't think there's any advantage to that versus continuous RX mode except it is more work.

That is weird that the timeout isn't canceled when it is receiving a packet like it should be though, so maybe 95% interval is fine.